### ISSUE TITLE
cross platform support

### ISSUE THREAD

**Issue by d4rkd0s** [brando5393/keyboardstoplog#7]
Currently I only see windows source code.

Do you have any plans to port to Mac OSX or Linux?

**Comment by brando5393** [comment]
Yes, I do plan to port the program when it is finished. I was thinking of coding it in python for OSX and Linux.
